docs/authenticate.js

```diff
@@ -276,6 +288,65 @@ function initializeFirebase() {
   firebase.initializeApp(getFirebaseConfig(inProduction()));
 }
 
+const allReadBinding = Object.freeze({
+  role: 'roles/viewer',
+  members: ['allUsers'],
+});
+
+/**
+ * Lists assets in the current disaster's folder and for any that match a score
+ * asset path (either standard or backup) and are not already world-readable,
+ * send a request to make that asset world-readable. This should only be called
+ * when logged in as the GD user. It does not wait for these calls to complete,
+ * and does not print any errors, since it is just trying to help other users,
+ * and is not triggered by an explicit user action.
+ *
+ * Optimized for the common case of assets already being world-readable: the
+ * getIamPolicy is technically not necessary, but usually it will avoid a set,
+ * which should be at least as expensive.
+ */
+function makeScoreAssetsWorldReadable() {
+  // TODO(janakr): Switch to using listEeAssets once #368 is in.
+  // TODO(janakr): Consider sharing cache with list_ee_assets.js. Not trivial
+  //  because that code does additional EE requests to look at geometries, so
+  //  we would need a two-level cache, one raw and one with geometries.
```

Comment:
I had the idea that you could pass in an extra filtering function to listEeAssets that does something like geometry filtering and would really generify listEeAssets. This is potentially compelling for that direction as well.

Reply:
Sure, we could do that, but there's still the question of how to cache. You'd have to cache keyed on filtering function. I think the simplest thing to do is probably just have two caches, one of the raw listings and one of the processed listings, but still not positive it's worth it.